multisigcore/providers/insight.py

```diff
@@ -23,11 +26,13 @@ def spendables_for_addresses(self, bitcoin_addresses):
             addresses = bitcoin_addresses[i:i+CHUNK_SIZE]
             url = "%s/api/addrs/%s/utxo" % (self.base_url, ",".join(addresses))
             r.extend(json.loads(urlopen(url).read().decode("utf8")))
-        spendables = []
+        spendables = {}
```

Comment:
kk great. Also, notice the `class InsightBatchService(InsightService, BatchService)`, without it, it was still pulling one by one (in my case, it took about 20 seconds :) )